client/packages/invoices/src/InboundShipment/DetailView/modals/InboundLineEdit/InboundLineEdit.tsx

```diff
@@ -19,17 +11,15 @@ import {
   useConfirmOnLeaving,
   TableProvider,
   createTableStore,
-  TabKeybindings,
   createQueryParamsStore,
   useKeyboardHeightAdjustment,
 } from '@openmsupply-client/common';
 import { ItemRowFragment } from '@openmsupply-client/system';
-import { InboundLineEditPanel } from './InboundLineEditPanel';
-import { QuantityTable, PricingTable, LocationTable } from './TabTables';
 import { InboundLineEditForm } from './InboundLineEditForm';
 import { useInbound } from '../../../api';
 import { DraftInboundLine } from '../../../../types';
 import { CreateDraft } from '../utils';
+import { TabLayout } from './TabLayout';
```

Comment:
I thought that the jsx in this file was getting too large, so lifted it out to a new component